piksi_tools/settings.py

```diff
@@ -317,8 +317,9 @@ def get_args():
                      "  2: Improper usage")
     parser.add_argument(
         "--timeout",
+        type=float,
         default=DEFAULT_TIMEOUT_SECS,
-        help="specify the timeout for settings reads.")
+        help="timeout for settings calls in s (default: %.1f)." % DEFAULT_TIMEOUT_SECS)
```

Comment:
`in seconds` or `timeout for settings calls [s]`

Reply:
clarified